cmake/Summary.cmake

```diff
@@ -81,7 +81,9 @@ function(caffe2_print_configuration_summary)
     message(STATUS "    USE_MEM_EFF_ATTENTION : ${USE_MEM_EFF_ATTENTION}")
     if(${USE_CUDNN})
       message(STATUS "    cuDNN version       : ${CUDNN_VERSION}")
+      message(STATUS "    USE_SYSTEM_CUDNN_FRONTEND : ${USE_SYSTEM_CUDNN_FRONTEND}")
     endif()
+    message(STATUS "    USE_SYSTEM_CUTLASS  : ${USE_SYSTEM_CUTLASS}")
```

Comment:
Likewise haven't done a great job of maintaining backward compatibility here for changes since it was previously pinned.